website/README.md

```diff
@@ -68,6 +80,12 @@ Before sending the PR for review, please run:
 
     $ git checkout -- content
```

Comment:
The `git checkout -- content` step is no longer necessary; this was to ensure generated content does not get committed, but this should now be covered in the .gitignore file.

Reply:
If you run Jekyll manually, it'll still create a `./generated-content` directory in the CWD. The Gradle task takes care to run under `build/website`.
I'll go ahead and remove the Jekyll commands to make this guide shorter and less  TLDR-ish.